src/메뉴 리뉴얼.js

```diff
@@ -0,0 +1,47 @@
+function solution(orders, course) {
+  const hash = {};
+
+  for (let i = 0; i < course.length; i++) {
+    let j = 0;
+    let k = 0;
+
+    while (k < orders.length) {
+      j = 0;
+
+      while (j < orders[k].length) {
+        // [2,3,4]
+        const menus = orders[k].substring(j, j + course[i]);
+        j++;
+
+        if (menus.length < course[i]) {
+          continue;
+        }
+
+        if (!hash[menus]) {
+          hash[menus] = 1;
+        } else {
+          hash[menus] = ++hash[menus];
```

Comment:
```suggestion
   hash[menus] = hash[menus] ? hash[menus] + 1 : 1;
```
삼항연산자를 이용하여 작성할 수도 있을 꺼 같습니다!